### PR TITLE
 Add reference to pause and resume methods in french doc (reference #32).

### LINKED ISSUE #32: French documentation of #30
Write french documentation for #30 
- new Bench methods
```php
    /**
     * Pause all marks.
     *
     * @return  array
     */
    public function pause();

    /**
     * Resume all marks.
     *
     * @param   array  $runningMarks    The marks to resume.
     * @return  void
     */
    public function resume($runningMarks);
```

### PR DESCRIPTION
**Pull request by osaris**
Edit from @Hywan: Fix #32.

Sorry but I didn't remember how to generate documentation for the current lib so I didn't test in my browser :-(

### CODE REVIEW

**Comment by Hywan** [comment]
@osaris Just run:
```sh
$ composer install
$ vendor/bin/hoa devtools:documentation
```
Does it work?

File: Documentation/Fr/Index.xyl [hunk 1]
```diff
@@ -114,6 +114,11 @@ echo $bench;
  *     __global__  ||||||||||||||||||||||||||||||||||||||||||||||||||||    45ms, 100.0%
  *     aMark       |||||||||||||                                           11ms,  24.8%
  */</code></pre>
+  <p>Nous pouvons également mettre en pause toutes les marques et récupérer
+  quelles marques étaient en cours d'exécution à ce moment grâce à la méthode
+  <code>Hoa\Bench\Bench::pause</code>. La méthode
+  <code>Hoa\Bench\Bench::resume</code> permet de redémarrer un ensemble de
+  marques.</p>
```
Comment (Hywan) [inline Documentation/Fr/Index.xyl]: Do you think we need an example here?

**Comment by osaris** [comment]
No, ```composer install``` doesn't install any binary.

File: Documentation/Fr/Index.xyl (hunk 1 above)
Comment (osaris) [inline Documentation/Fr/Index.xyl]: These methods are useful only in some specifics cases nope ? So I don't think we need an example.

**Comment by Hywan** [comment]
@osaris Correct, that's because `hoa/test` is not included yet.
```sh
$ composer require hoa/test
$ vendor/bin/hoa devtools:documentation --language fr
```

**Comment by osaris** [comment]
Well it generates documentation for all libs in ```vendor/``` but not for the current lib :-(

**Comment by Hywan** [comment]
@osaris Just tested and it works:
```sh
$ mkdir -p /tmp/bench
$ cd !!$
$ composer require hoa/bench
$ composer require hoa/devtools
$ vendor/bin/hoa devtools:documentation --language fr
Selected language: Fr.

Generate        index.html.
Generate        Hoa\Bench.
Generate        Hoa\Compiler.
Generate        Hoa\Console.
Generate        Hoa\Dispatcher.
Generate        Hoa\Router.
Generate        Hoa\Ustring.
Generate        Hoa\View.

Open file:///tmp/bench/vendor/hoa/devtools/Resource/Documentation/HackBook.output/Fr/index.html.
```
See? We have `Hoa\Bench` :-).

**Comment by osaris** [comment]
You are requiring Hoa\Bench so you are not in the Hoa\Bench lib folder ?!

2015-09-11 15:28 GMT+02:00 Ivan Enderlin <notifications@github.com>:

**Comment by Hywan** [comment]
@osaris Correct. I thought it was fixed… Go like this and edit the `vendor/hoa/bench` directory. And please, file an issue on `Hoa\Devtools` :-).

**Comment by osaris** [comment]
```vendor/bin/hoa devtools:documentation --language fr -d .``` did the trick !

**Comment by osaris** [comment]
Now waiting for a review to do the same modification in english doc ;-)

File: Documentation/Fr/Index.xyl (hunk 1 above)
Comment (Hywan) [inline Documentation/Fr/Index.xyl]: Agree :-].

**Comment by Hywan** [comment]
Thanks!

**Pull request merged**